packages/bridge-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Bump `@metamask/assets-controllers` peer dependency to `^63.1.0` ([#5850](https://github.com/MetaMask/core/pull/5850))
```

Comment:
Nit: we dont need this bump

Reply:
Yep okay.

I added this since CI workflow said that the bridge-controller was modified and needed a changelog entry. I'll use the "no changelog" label.

Reply:
@cryptodev-2s updated in https://github.com/MetaMask/core/pull/5850/commits/179e877c01c5dd5673271f0014ad3ac32b104bbf